OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -878,9 +838,10 @@ calcMuscleDynamicsInfo(const SimTK::State& s, MuscleDynamicsInfo& mdi) const
         // Compute dynamic quantities.
         double a = SimTK::NaN;
         if(!get_ignore_activation_dynamics()) {
-            a = clampActivation(getStateVariableValue(s, STATE_ACTIVATION_NAME));
+            a = getActivationModel().clampActivation(
+                    getStateVariableValue(s, STATE_ACTIVATION_NAME));
```

Comment:
I agree. Please read the "Exclusions" paragraph at the top. I have already built a state-owning activation dynamics model so, in my opinion, it would be a waste of time to modify the old model to own the activation state. In my opinion, the next logical step is to swap in the new model.